### PR TITLE
Documentation(822850): To include the [FromForm] attribute and how to import an excel through uploader success event

### PR DESCRIPTION
**Pull request by sangeetha-priya-M**
Feature description
Need to include the [FromForm] attribute in open/save documentation and added content for how to import a file in spreadsheet thorough uploader success event

Analysis and design
Included the  [FromForm] attribute in open/save documentation and added content for how to import a file in spreadsheet thorough uploader success event
Solution description
Included the  [FromForm] attribute in open/save documentation and added content for how to import a file in spreadsheet thorough uploader success event

Output Screenshot
NA

Areas affected and ensured
NA

Test cases
NA

### CODE REVIEW

File: ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+platform: ej2-javascript
+control: Spreadsheet 
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+## Import an excel file using uploader component in ##Platform_Name## Spreadsheet control
+
+If you select the file using file uploader then you will get the uploaded file data in the [`success`](../api/uploader#success) event of file uploader. In this event, you should pass the received file data as argument to the [`open`](../api/spreadsheet/#open) method of the spreadsheet.
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md:14]: Don't highlight the API names and methods

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+ej.base.enableRipple(true);
+
+var spreadsheet = new ej.spreadsheet.Spreadsheet({
+  openUrl: 'https://services.syncfusion.com/js/production/api/spreadsheet/open',
+});
+
+//Render initialized Spreadsheet component
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.js:7]: Remove the unwanted comment lines

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { Spreadsheet } from '@syncfusion/ej2-spreadsheet';
+import { Uploader } from '@syncfusion/ej2-inputs';
+//Initialize the SpreadSheet control
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts:3]: Remove the unwanted comment lines

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+let spreadsheet: Spreadsheet = new Spreadsheet({
+  openUrl: 'https://services.syncfusion.com/js/production/api/spreadsheet/open',
+});
+
+spreadsheet.appendTo('#spreadsheet');
+
+// Initialize the uploader component
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts:10]: Remove the unwanted comment lines

**Review by gobalakrishnans — CHANGES_REQUESTED** [review]

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/system.config.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        main: "index.ts",
+        typescript: "https://unpkg.com/typescript@2.2.2/lib/typescript.js",
+        //Syncfusion packages mapping
+        "@syncfusion/ej2-base": "syncfusion:ej2-base/dist/ej2-base.umd.min.js",
+        "@syncfusion/ej2-data": "syncfusion:ej2-data/dist/ej2-data.umd.min.js",
+        "@syncfusion/ej2-inputs": "syncfusion:ej2-inputs/dist/ej2-inputs.umd.min.js",
+        "@syncfusion/ej2-buttons": "syncfusion:ej2-buttons/dist/ej2-buttons.umd.min.js",
+        "@syncfusion/ej2-splitbuttons": "syncfusion:ej2-splitbuttons/dist/ej2-splitbuttons.umd.min.js",
+        "@syncfusion/ej2-lists": "syncfusion:ej2-lists/dist/ej2-lists.umd.min.js",
+        "@syncfusion/ej2-navigations": "syncfusion:ej2-navigations/dist/ej2-navigations.umd.min.js",
```
[the remark this answers is not in the document]
Comment (sangeetha-priya-M) [reply ej2-javascript/code-snippet/spreadsheet/import-using-uploader/system.config.js:25]: Added the package

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts (hunk 4 above)
Comment (sangeetha-priya-M) [reply ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts:10]: removed

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts (hunk 3 above)
Comment (sangeetha-priya-M) [reply ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.ts:3]: removed

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.js (hunk 2 above)
Comment (sangeetha-priya-M) [reply ej2-javascript/code-snippet/spreadsheet/import-using-uploader/index.js:7]: removed

File: ej2-javascript/spreadsheet/ej2-typescript-spreadsheet-toc.html [hunk 6]
```diff
@@ -40,6 +40,7 @@
 <li><a href="/ej2-typescript/spreadsheet/how-to/create-a-object-structure">Create a JSON structure</a></li>
 <li><a href="/ej2-typescript/spreadsheet/how-to/print">Print the single multiple sheets</a></li>
 <li><a href="/ej2-typescript/spreadsheet/how-to/change-active-sheet">Changing the active sheet while importing a file</a></li>
+<li><a href="/ej2-typescript/spreadsheet/how-to/import-file-using-uploader">Import an excel file using uploader component</a></li>
```
[the remark this answers is not in the document]
Comment (sangeetha-priya-M) [reply ej2-javascript/spreadsheet/ej2-typescript-spreadsheet-toc.html:43]: modified

File: ej2-javascript/code-snippet/spreadsheet/import-using-uploader/system.config.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (sangeetha-priya-M) [reply ej2-javascript/code-snippet/spreadsheet/import-using-uploader/system.config.js:25]: modified

File: ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Import an excel file using uploader component in ##Platform_Name## Spreadsheet control | Syncfusion
```
[the remark this answers is not in the document]
Comment (sangeetha-priya-M) [reply ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md:3]: modified

File: ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+description: Learn here all about import an excel file using uploader in Syncfusion ##Platform_Name## Spreadsheet control of Syncfusion Essential JS 2 and more.
+platform: ej2-javascript
+control: Spreadsheet 
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+## Import an excel file using uploader component in ##Platform_Name## Spreadsheet control
```
[the remark this answers is not in the document]
Comment (sangeetha-priya-M) [reply ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md:12]: modified

File: ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md (hunk 1 above)
Comment (sangeetha-priya-M) [reply ej2-javascript/spreadsheet/how-to/import-file-using-uploader.md:14]: modified

**Review by gobalakrishnans — APPROVED** [review]

**Pull request merged**